### DISCUSSION TITLE
Place-able Pumpkins/End Crystals

### DISCUSSION THREAD

**Discussion by Lylythii** [ACGaming/UniversalTweaks#97] [Tweak Suggestions]
- Pumpkins require a block below them for no real reason.

- End Crystals require obsidian or bedrock to be place-able.

Neither are desire-able!

Existing mods that do this;
- https://www.curseforge.com/minecraft/mc-mods/placeableendcrystals
- https://www.curseforge.com/minecraft/mc-mods/placepumpkinanywhere

**Comment by Turkey9002** [comment]
[Subaquatic](https://www.curseforge.com/minecraft/mc-mods/subaquatic) allows you to place pumpkins anywhere if you have that installed (while changing the pumpkins to be faceless unless they're carved with shears like in 1.13+). It also backports some Update Aquatic content and the boat chests from The Wild Update.

**Comment by WaitingIdly** [comment]
implemented in #425 and #428.
